### DISCUSSION TITLE
osu!lazer crashes right after opening

### DISCUSSION THREAD

**Discussion by ohSchy** [ppy/osu#16470] [Q&A] [answered]
[network.log](https://github.com/ppy/osu/files/7876446/network.log)
[performance.log](https://github.com/ppy/osu/files/7876447/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7876448/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7876449/updater.log)
[database.log](https://github.com/ppy/osu/files/7876450/database.log)

when osu!lazer opens, it immediately crashes, I also had the same issue in the past (1-2 months ago), but it went away randomly without any changes
I've seen this mentioned by others here also where they've just randomly ended the discussion by "oh it's fixed" but there's never been a specific answer to this problem
sorry if i missed something important, first time I've posted something here

**Comment by peppy** [comment, marked as answer]
There's no error in your logs, so it's likely something else on your PC causing the crash. You'll have to check windows event viewer and figure it out from there.
